specification/cognitiveservices/data-plane/ComputerVision/stable/v3.2/Ocr.json

```diff
@@ -594,6 +603,15 @@
       "type": "string",
       "pattern": "^(latest|\\d{4}-\\d{2}-\\d{2})(-preview)?$",
       "default": "latest"
+    },
+    "ReadingOrder": {
+      "name": "readingOrder",
+      "in": "query",
+      "description": "Optional parameter to specify which reading order algorithm should be applied when ordering the extract text elements. Can be either 'basic' or 'natural'. Will default to 'basic' if not specified",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "default": "basic"
```

Comment:
Autorest has a complex story with `default`, so I want to be sure I explain all the design flaws and questionable behavior so there is no surprise:

`default` : means in Swagger "the server will do that if client doesn't provide the value". It is NOT intended that client put that on the call. However, autorest track1 (the generation you are still using) incorrectly handle this value and will send explicity "readingOrder=basic" by default if you specify `default`. If this is a problem, and you really care that the default call does not include `readingOrder`, then I have no other solution today than to ask you to remove `default` from this Swagger. If you don't care, I would encourage to keep `default` since this makes the Swagger more accurate.